hack/packages/package-tools/cmd/vendir-sync.go

```diff
@@ -43,40 +44,50 @@ func runPackageVendirSync(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	var g errgroup.Group
```

Comment:
I know we Go programmers try to keep variables as short as possible, but the caveat is that it needs to be clear what it means. I think `g` is used far enough from its declaration, it should be named something like `errGrp` to be clear.